lib/package.gi

```diff
@@ -1124,7 +1137,7 @@ InstallGlobalFunction( DefaultPackageBannerString, function( inforec )
     # Add package name and version number.
     if IsBound( inforec.PackageName ) and IsBound( inforec.Version ) then
       Append( str, Concatenation(
-              "Loading  ", inforec.PackageName, " ", inforec.Version ) );
+              "Loading ", inforec.PackageName, " ", inforec.Version ) );
```

Comment:
I am somewhat wary that this might break package tests. Although a quick look suggests that this is not the case, so perhaps all is good in that regard.

I can't think of a good reason for the extra space otherwise.

Reply:
I have now (hopefully) fixed the tests in GAP. Maybe we can merge and see how many packages are affected by this, if any at all? If the number is small I could open PRs, otherwise we could revert.